conversiontracerbullet/src/main/java/org/stanford/AuthDBProperties.java

```diff
@@ -19,22 +16,22 @@
 
     private static Logger log = LogManager.getLogger(AuthDBProperties.class.getName());
 
-    private static final String DEFAULT_CONFIG_FILE = "/server.conf";
+    private static final String PROPERTY_RESOURCE = "/server.conf";
 
     private String server = null;
     private String service = null;
     private String userName = null;
     private String userPass = null;
-    private String propertyFile = null;
+    private Properties properties = null;
```

Comment:
If I get your point correctly, it seems your requesting changes like the following:
```java
            // lines 84 - 85 are:
           Properties props = new Properties();
           props.load(iStream);
           // they are replaced by using the instance var
           properties.load(iStream);
```

I'll try that and see what happens.